packages/editor-ui/src/api/cloudPlans.ts

```diff
@@ -0,0 +1,34 @@
+import type { CloudPlanData, IRestApiContext } from '@/Interface';
+
+export function getCurrentPlan(
+	context: IRestApiContext,
+	cloudUserId: string,
+): Promise<CloudPlanData> {
+	return Promise.resolve({
+		planSpec: {
+			planId: 43039,
+			monthlyExecutionsLimit: 200,
+			activeWorkflowsLimit: 10,
+			credentialsLimit: 100,
+			isActive: false,
+			displayName: 'Trial',
+			expirationDate: '2023-05-06T01:47:47Z',
+			metadata: {
+				version: 'v1',
+				group: 'opt-in',
+				slug: 'trial-1',
+				trial: {
+					length: 7,
+					gracePeriod: 3,
+				},
+			},
+		},
+		instance: {
+			createdAt: '2023-05-01T01:47:47Z',
```

Comment:
do we still need this? anyways, if we are going to have an `expirationDate`, maybe we need then `startDate` next to it which would be more clear..

Reply:
Not needed anymore, so I removed it.